packages/event-listener/test/index.test.ts

```diff
@@ -1,76 +1,99 @@
-import { suite } from "uvu";
+import { dispatchFakeEvent } from "./setup";
+import { test } from "uvu";
 import * as assert from "uvu/assert";
-import { createRoot, createSignal } from "solid-js";
-import { createEventListener, EventListenerProps } from "../src/index";
+import { createRoot, createSignal, onMount } from "solid-js";
+import { createEventListener, EventListenerDirectiveProps } from "../src";
 
-const test = suite("createEventListener");
+test("it will add an event", () => {
```

Comment:
Removing the suite means that you no longer see the name of the tested function.